explore-analyze/visualize/charts/metric-charts.md

```diff
@@ -0,0 +1,294 @@
+::::
+
+::::{step} Customize the chart with best practices
+Tweak the appearance of the chart to your needs. Consider the following best practices:
+
+**Use color wisely**
+:   Assign colors that match your users' expectations and consider your specific context: for costs, lower might be better, while for revenue, higher is better.
+
+**Format for readability**
+:   Round to appropriate precision. Showing 1.2M is clearer than 1,234,567.89 for high-level metrics. But show more precision when small changes matter.
+
+**Provide context**
+:   Use titles and subtitles to explain what the metric means. "45,234" is a number, but "Active Users" as a title gives it meaning, and "Last 24 hours" as a subtitle makes it unambiguous.
+
+Refer to [](#settings) for a complete list of options.
+::::
+
+::::{step} Save the chart
+Save your visualization by selecting **Save and return** if you're adding it from a dashboard, or **Save to library** to use it later.
+::::
+
+:::::
+
+## Advanced metric scenarios
+
+### Show trends in Metric charts [metric-trends]
+```{applies_to}
+stack: ga 9.1
+serverless: ga
+```
+
+When creating **Metric** visualizations with numeric data, you can add trend indicators that compare your primary metric to a secondary value. This feature displays colored badges with directional arrows to help you quickly identify whether values are increasing, decreasing, or staying the same.
+
+| Without trend | With trend |
+|--------|-------|
+| ![Secondary metric before comparison](../../images/secondary_metric_before_compare.png "title =70%") | ![Secondary metric after comparison](../../images/secondary_metric_after_compare.png "title =70%") |
+
+To add trend indicators to your metric visualization:
+
+1. Create a **Metric** visualization with a numeric primary metric.
+2. Add a secondary metric that represents the comparison value.
+
+    ::::{tip}
+    Use the `shift` parameter in formulas to compare current values against historical data. For example, if your primary metric is counting orders (based on an `order_id` field) for the current week, you can use the `count(order_id, shift='1w')` formula to compare this week's count of orders to last week's count.
+    ::::
+
+3. In the secondary metric configuration, look for the **Color by value** option. The possible choices are:
+   * **None**: No trend indicators (default)
+   * **Static**: Shows the secondary metric as a badge with a fixed color that you select
+   * **Dynamic**: Enables both color coding and directional icons based on the comparison
+
+4. Select **Dynamic** coloring. More options appear.
+
+   ![Dynamic coloring options for secondary metrics](../../images/metric-secondary-dynamic-coloring.png "=70%")
+
+5. Choose a **Color palette** that matches how you'd like to represent the comparison.
+
+6. Configure the **Display** option:
+   * **Icon**: Shows only directional arrows: ↑ for increase, ↓ for decrease, = for no change
```

Comment:
I noticed that there's EUI [icons](https://eui.elastic.co/docs/components/display/icons/) for the increase and decrease arrows, but they seem to be used for sorting (sortDown and sortUp). Should the EUI icons be used here? 

Unfortunately, I couldn't find an icon for the equal sign (`=`).